### PR TITLE
Updating docstrings to prevent indentation errors in sphinx

### PR DESCRIPTION
**Pull request by stephanielee9**

### CODE REVIEW

**Comment by pgoiporia** [comment]
Sorry I haven't been able to get to this yet! Tagging myself so that I remember to review later today. @pgoiporia

File: cirq_superstaq/service.py [hunk 1]
```diff
@@ -363,8 +363,8 @@ def aqt_compile_eca(
             object whose .circuits attribute is a list (or list of lists) of logically equivalent
                 cirq Circuit(s).
             If qtrl is installed, the object's .seq attribute is a qtrl Sequence object of the
-            pulse sequence corresponding to the cirq.Circuits and the .pulse_lists attribute is
-            the list(s) of cycles.
+                pulse sequence corresponding to the cirq.Circuits and the .pulse_lists attribute is
```
Comment (pgoiporia) [inline cirq_superstaq/service.py:366]: I believe we use spaces over tabs
```suggestion
                pulse sequence corresponding to the cirq.Circuits and the .pulse_lists attribute is
```

Comment (pgoiporia) [reply cirq_superstaq/service.py:366]: I don't know if github detects the change I suggested (I replaced the tab with 4 spaces). If not, you can just change it locally.

File: cirq_superstaq/service.py [hunk 2]
```diff
@@ -363,8 +363,8 @@ def aqt_compile_eca(
...
+                the list(s) of cycles.
```
Comment (pgoiporia) [inline cirq_superstaq/service.py:367]: Same as above

File: cirq_superstaq/custom_gates.py [hunk 3]
```diff
@@ -311,15 +317,15 @@ class ParallelGates(cirq.Gate, cirq.InterchangeableQubitsGate):
     """A single Gate combining a collection of concurrent Gate(s) acting on different qubits.
 
     WARNING: for cirq versions 0.14.*, equality check will return False after permutations of
-        qubits between identical but nonadjacent gates, e.g.:
+        qubits between identical but nonadjacent gates, e.g.::
```
Comment (pgoiporia) [inline cirq_superstaq/custom_gates.py:320]: Not a mistake just something I'm curious about. What exactly does :: denote. Is it for code blocks or for general indentation? Or something else entirely.

**Review by pgoiporia — CHANGES_REQUESTED** [review]
Nice work! Just a couple of quick fixes.
I'm assuming you've made these changes to fix something in docs-superstaq. Have you tested whether they work?

File: cirq_superstaq/custom_gates.py (hunk 3 above)
Comment (stephanielee9) [reply cirq_superstaq/custom_gates.py:320]: Yep! `::` is for code blocks in reStructuredText

**Comment by stephanielee9** [comment]
> Have you tested whether they work?

Yep! I rebuilt the docs in docs-superstaq and these changes removed the errors for me

File: cirq_superstaq/service.py (hunk 1 above)
Comment (pgoiporia) [reply cirq_superstaq/service.py:366]: @vtomole looks like this file uses tabs, but when Stephanie changes it to spaces, git doesn't see any diff

**Comment by pgoiporia** [comment]
> > Have you tested whether they work?
> 
> Yep! I rebuilt the docs in docs-superstaq and these changes removed the errors for me

Awesome! lgtm!

**Review by pgoiporia — APPROVED** [review]

**Pull request merged**